dataset_config/platinum_genomes/ui.json

```diff
@@ -0,0 +1,18 @@
+// Configuration for Data Explorer UI
```

Comment:
This config seems like it should exist in the Data Explorer repo. While I understand that it's easy to copy a config folder from one repo to another, the contents of those folders are now sufficiently distinct that it makes sense to just have them separate.

Reply:
I disagree. It's easier for dataset owner to maintain one set of config files per dataset, than two.

Say someone is maintaining 10 Commons datasets. It's easier to maintain 10 directories versus 20. And the 20-directory case would be complicated by the fact that some config files (dataset.json) are in two directories, and have to be maintained in sync.

Reply:
Also, note that the files are related. If someone wants to change add a BQ table, they would have to edit both bigquery.json and ui.json. It's easier if the files are in the same directory, versus they have to think about which directory the files are in.

Reply:
Done, removed ui.json